docs/en/Architecture-DataFlow.md

```diff
@@ -32,23 +33,40 @@ When mounting multiple under storage systems simultaneously, Alluxio serves as a
 layer for any number of varied data sources.
 
 Alluxio can be divided into three components: masters, workers, and clients.
-A typical setup consists of a single leading master, multiple standby masters,
-and multiple workers. The master and worker processes constitute the Alluxio servers,
-which are the components a system administrator would maintain. The clients are used to
-communicate with the Alluxio servers by applications such as Spark or MapReduce jobs,
-Alluxio command-line, or the FUSE layer.
+A typical setup consists of a single leading master, multiple standby masters, a job master,
```

Comment:
remove `multiple` since we don't say `multiple workers`? Also are there standby job masters?

Reply:
fixed and added standby job masters